addons/mail/static/tests/tours/discuss_mention_suggestions_group_restricted_channel.js

```diff
@@ -0,0 +1,24 @@
+import { registry } from "@web/core/registry";
+import { contains } from "@web/../tests/utils";
+
+registry.category("web_tour.tours").add("discuss_mention_suggestions_group_restricted_channel.js", {
+    steps: () => [
+        { trigger: ".o-mail-Discuss-threadName[title='R&D Channel']" },
+        { trigger: ".o-mail-Composer-input", run: "edit @" },
+        {
+            trigger: ".o-mail-Composer-suggestionList",
+            async run() {
+                // Suggestions: Consultant User, Mitchell Admin, Dev User
+                await contains(".o-mail-Composer-suggestion", { count: 3 });
```

Comment:
`:count()` should now be supported in selectors, we should no longer use contains in run steps